turbinia/workers/partitions.py

```diff
@@ -118,25 +126,35 @@ def run(self, evidence, result):
     Returns:
       TurbiniaTaskResult object.
     """
-    result.log('Scanning [{0:s}] for partitions'.format(evidence.local_path))
+    evidence_description = None
```

Comment:
I filed a new issue so that we can get meaningful evidence names here: https://github.com/google/turbinia/issues/718 .  Could you maybe add a TODO here referencing that so we can remove these once it gets fixed there?